src/17D6A0.c

```diff
@@ -2387,6 +2405,133 @@ s32 bActorMessages[] = {
+    gsSPClearGeometryMode(G_SHADING_SMOOTH),
+    gsSPVertex(D_80283B20, 4, 0),
+    gsSP1Triangle(0, 1, 2, 0),
+    gsSP1Triangle(0, 2, 3, 0),
+    gsSPEndDisplayList(),
+};
+
+EntityModelScript D_80283C48 = STANDARD_ENTITY_MODEL_SCRIPT(D_80283B60, RENDER_MODE_ALPHATEST);
+
+Vtx D_80283C68[4] = {
+    {{{ -16, -16, 0 }, 0, {    0,    0 }, { 0, 0, 0, 255 }}},
+    {{{  15, -16, 0 }, 0, { 1024,    0 }, { 0, 0, 0, 255 }}},
+    {{{  15,  15, 0 }, 0, { 1024, 1024 }, { 0, 0, 0, 255 }}},
+    {{{ -16,  15, 0 }, 0, {    0, 1024 }, { 0, 0, 0, 255 }}},
+};
+
+Gfx D_80283CA8[] = {
+    gsDPPipeSync(),
+    gsSPTexture(0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_ON),
+    gsDPSetCombineLERP(0, 0, 0, TEXEL0, PRIMITIVE, 0, TEXEL0, 0, 0, 0, 0, TEXEL0, TEXEL0, 0, PRIMITIVE, 0),
+    gsDPSetTexturePersp(G_TP_PERSP),
+    gsDPSetTextureDetail(G_TD_CLAMP),
+    gsDPSetTextureLOD(G_TL_TILE),
+    gsDPSetTextureLUT(G_TT_NONE),
+    gsDPSetTextureFilter(G_TF_AVERAGE),
+    gsDPSetTextureConvert(G_TC_FILT),
+    gsDPSetTextureLUT(G_TT_RGBA16),
+    gsDPLoadTLUT_pal16(0, ui_battle_menu_spirits_pal),
+    gsDPLoadTextureTile_4b(ui_battle_menu_spirits_png, G_IM_FMT_CI, 32, 0, 0, 0, 31, 31, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD),
+    gsSPClearGeometryMode(G_LIGHTING),
+    gsSPClearGeometryMode(G_SHADING_SMOOTH),
+    gsSPVertex(D_80283C68, 4, 0),
+    gsSP1Triangle(0, 1, 2, 0),
+    gsSP1Triangle(0, 2, 3, 0),
+    gsDPPipeSync(),
+    gsSPEndDisplayList(),
+};
+
+EntityModelScript D_80283D98 = STANDARD_ENTITY_MODEL_SCRIPT(D_80283CA8, RENDER_MODE_2D);
+
+Vtx D_80283DB8[4] = {
+    {{{ -16, -16, 0 }, 0, {    0, 1024 }, { 0, 0, 0, 255 }}},
+    {{{  15, -16, 0 }, 0, { 1024, 1024 }, { 0, 0, 0, 255 }}},
+    {{{  15,  15, 0 }, 0, { 1024,    0 }, { 0, 0, 0, 255 }}},
+    {{{ -16,  15, 0 }, 0, {    0,    0 }, { 0, 0, 0, 255 }}},
+};
+
+Gfx D_80283DF8[] = {
+    gsDPPipeSync(),
+    gsSPTexture(0xFFFF, 0xFFFF, 0, G_TX_RENDERTILE, G_ON),
+    gsDPSetCombineMode(G_CC_DECALRGBA, G_CC_DECALRGBA),
+    gsDPSetTexturePersp(G_TP_PERSP),
+    gsDPSetTextureDetail(G_TD_CLAMP),
+    gsDPSetTextureLOD(G_TL_TILE),
+    gsDPSetTextureLUT(G_TT_NONE),
+    gsDPSetTextureFilter(G_TF_AVERAGE),
+    gsDPSetTextureConvert(G_TC_FILT),
+    gsDPSetTextureLUT(G_TT_RGBA16),
+    gsDPLoadTLUT_pal16(0, ui_battle_unk_star_pal),
+    gsDPLoadTextureTile_4b(ui_battle_unk_star_png, G_IM_FMT_CI, 32, 0, 0, 0, 31, 31, 0, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMIRROR | G_TX_WRAP, G_TX_NOMASK, G_TX_NOMASK, G_TX_NOLOD, G_TX_NOLOD),
+    gsSPClearGeometryMode(G_LIGHTING),
+    gsSPClearGeometryMode(G_SHADING_SMOOTH),
+    gsSPVertex(D_80283DB8, 4, 0),
+    gsSP1Triangle(0, 1, 2, 0),
+    gsSP1Triangle(0, 2, 3, 0),
+    gsDPPipeSync(),
+    gsSPEndDisplayList(),
+
+};
+EntityModelScript D_80283EE8 = STANDARD_ENTITY_MODEL_SCRIPT(D_80283DF8, RENDER_MODE_ALPHATEST);
+
+s32 D_80283F04_padding = 0;
+s32 D_80283F08_padding = 0;
+s32 D_80283F0C_padding = 0;
```

Comment:
Perhaps make these `static`?
@ethteck I'm thinking we should maybe make a `DATA_PADDING(addr, size)` macro for this purpose

Reply:
I would rather just keep the padding temps personally

Reply:
explicit padding temps*

Reply:
That's fine, was just suggesting so they're easier to find and clean up later but tbh we can just look for "pad" and find them all